src/Blocks/BlockCrops.h

```diff
@@ -81,9 +43,51 @@ class cBlockCropsHandler :
 					break;
 				}
 			}
+			return;
 		}
+
+		// Fully grown, drop the produce:
```

Comment:
`produce` is correct, but it sounds weird because it's an uncountable noun (like `sheep`).  `product`  could work, but it's more general ([Wiktionary](https://en.wiktionary.org/wiki/produce#Noun) lists it as a hypernym).

Perhaps `crop's produce` would be better?